doc/tests/manual/reports/231014.md

```diff
@@ -0,0 +1,142 @@
+# Manual Test Report (v0.3.4)
+
+## General
+
+- Tester: Aydan Namdar Ghazani
+- Date/Time: 14. October 2023
+- Duration: ~5 minutes
+- Commit/Tag: [1c3b54a](https://github.com/ElektraInitiative/PermaplanT/tree/1c3b54aa80c2fff938d7f9fe8cd662084c876873)
+- Setup: dev.permaplant.net
+- Planned tests: **11**
+- Executed tests: **8**
+- ✔️Passed tests: **5**
+- ⚠️Problematic tests: **3**
+- ❌Failed tests: **0**
+- Skipped tests: **6**
+
+## [Error Analysis](../README.md#report-header)
+
+No errors occured during the test.
+
+## Closing remarks
+
+### How is the current state of the software?
+
+The software is in a good state.
+
+### Have the quality objectives been achieved?
+
+There were no defined quality objectives at this time.
+
+### What are the consequences drawn from the current state, including: how can future errors be avoided, how can the development process be improved?
+
+We should remove tests from the protocol which are not implemented yet (Heatmap, Chat)
+
+## Testcases
+
+## Plant Search
+
+- Description: Show a selection of plants if the search input is empty.
+- Given I am on a map page with the plant layer active
+- When I have an empty search box
+- Then I can see various plants as results
+- Test Result: ✔️
+
+## Heatmap
+
+- Description: Test whether the heatmap endpoints generates the image correctly.
+- Given I am on a map page with the plant layer active
+- When I start planting a plant
+- Then I see suitable places for that plant
+- Test Result: Not Implemented
+
+## Base Layer
+
+- Description: Check whether the maps background image is displayed correctly.
+- Given I am on a map page with the base layer active
+- When I select a base layer image
+- Then I can see the base layer image on the canvas
+- Test Result: ✔️
+
+## Grid
+
+- Description: Display a point grid on the screen.
+- Given I am on a map page
+- When I Zoom in or out
+- Then the grid spacing is changing
+- Test Result: ✔️
+
+## Map Editor Guided Tour
+
+- Description: Check whether the Guided Tour leaves the Map Editor in its original state.
+- Given I am on a map page
+- When I do the Guided Tour
+- Then after I have done the Guided Tour the map is the same as before
+- Actual Result:
+- Test Result: ⚠️
+- Notes: Was not able to test, since my user already performed the tour in the past
```

Comment:
Can you create an extra user for the tour? (And never finish the tour for that user?)

Sometimes bugs in the tour are introduced after all...

Reply:
The login is cached and I can not make a new account anymore. Need to open a new browser, maybe thats an usability problem?

Reply:
I created a new user for that, it would be nice to also use him in e2e tests, but he would complete the tour there 100%.